tests/test_file_filter.py

```diff
@@ -87,6 +87,42 @@ def test_get_file_filter_terms():
     assert filter_terms == ["file 1\\.org", "/path/to/dir/.*\\.org"]
 
 
+def test_exclude_file_filter_terms():
```

Comment:
A more appropriate name for this unit test maybe `test_include_and_exclude_file_filter_terms`
```suggestion
def test_include_and_exclude_file_filter_terms():
```